src/databricks/labs/ucx/cli.py

```diff
@@ -165,5 +165,48 @@ def revert_migrated_tables(w: WorkspaceClient, schema: str, table: str, *, delet
         tm.revert_migrated_tables(schema, table, delete_managed=delete_managed)
 
 
+@ucx.command
+def migrate_uc_to_uc(
+    w: WorkspaceClient,
+    from_catalog: str,
+    from_schema: str,
+    from_table: str,
+    to_catalog: str,
+    to_schema: str,
+):
+    """move a uc table/tables from one schema to another schema in same or different catalog"""
+    logger.info("Running move command")
+    installation_manager = InstallationManager(w)
+    installation = installation_manager.for_user(w.current_user.me())
+    if not installation:
+        logger.error(CANT_FIND_UCX_MSG)
+        return
+    sql_backend = StatementExecutionBackend(w, installation.config.warehouse_id)
+    tmp = TableMapping(w, sql_backend)
+    tables = TablesMigrate(
+        TablesCrawler(sql_backend, installation.config.inventory_database),
+        w,
+        sql_backend,
+        tmp,
+    )
+    if from_catalog == "" or to_catalog == "":
+        logger.error("Please enter from_catalog and to_catalog details")
+        return
+    if from_schema == "" or to_schema == "" or from_table == "":
+        logger.error("Please enter from_schema, to_schema and from_table(enter * for migrating all tables) details.")
+        return
+    if from_catalog == to_catalog and from_schema == to_schema:
+        logger.error("please select a different schema or catalog to migrate to")
+        return
+    logger.error(f"migrating tables {from_table} from {from_catalog}.{from_schema} to {to_catalog}.{to_schema}")
+    tables.move_migrated_tables(
```

Comment:
What if tables are not migrated?

Reply:
@nfx  The calling method tables.move_migrated_tables should throw the relevant error messages and info of tables migrated. should we need to add a try catch here too in the cli?